### ISSUE TITLE
Addition of pyproject.toml is missing a proper version

### ISSUE THREAD

**Issue by OndrejZapletal** [CJWorkbench/channels_rabbitmq#30]
I've tried to use the latest version of this package (`1.3.0`), but during installation from from PyPi using pip it fails:
```
pip install --upgrade channels-rabbitmq
Requirement already satisfied: channels-rabbitmq in /home/ondrej_zapletal/.pyenv/versions/3.8.5/envs/alfred/lib/python3.8/site-packages (0.0.0)
Collecting channels-rabbitmq
  Using cached channels_rabbitmq-0.0.0-py3-none-any.whl
ERROR: Requested channels-rabbitmq from file:///home/ondrej_zapletal/.cache/pip/wheels/cc/c2/f6/2b3f5956bdcad3765c4f19d1a6bf9179f27c6ff1931b0d5ec4/channels_rabbitmq-0.0.0-py3-none-any.whl has different version in metadata: '0.0.0'
```
I'm guessing that it has to do with your recent addition of `pyproject.toml`. Suggested change:
```
[tool.poetry]
version = "1.3.0"
```
But I might be mistaken.

**Comment by wang8zai** [comment]
Agreed. Just having exact same issue. @adamhooper

**Comment by adamhooper** [comment]
Thanks for the report! I will fix Tuesday as per
https://nicholasnadeau.me/post/2020/8/one-version-to-rule-them-all-keeping-your-python-package-version-number-in-sync-with-git-and-poetry/

... For speedier delivery, pull requests are welcome :)

**Comment by wang8zai** [comment]
#31

**Comment by adamhooper** [comment]
Finally fixed the installation, on v1.3.1. Sorry for the wait!
